build.gradle

```diff
@@ -64,8 +64,9 @@ dependencies {
     implementation 'com.squareup.okhttp3:logging-interceptor:3.9.1'
     implementation 'com.fasterxml.jackson.core:jackson-databind:2.10.0.pr3'
     implementation 'commons-codec:commons-codec:1.13'
+    implementation 'com.auth0:java-jwt:3.10.3'
 
-
+    testImplementation 'com.auth0:jwks-rsa:0.11.0'
```

Comment:
Used for tests implementing `PublicKeyProvider` with `JwkProvider`

Reply:
Actually I don't know that a mock (or a dependency on `jwks-rsa`) is needed at all, since we load the keys from the test pem files. I'll update that.